### ISSUE TITLE
no pdb outcome in the pyrosetta run version

### ISSUE THREAD

**Issue by ajian-he** [RosettaCommons/RoseTTAFold#50]
runs normal, however, there  is no pdb file out in the result folder

$ run_pyrosetta_ver.sh XXX.fasta .
Running HHblits
Running PSIPRED
Running hhsearch
Predicting distance and orientations
Running parallel RosettaTR.py
$ ls
 hhblits  log  parallel.fold.list  pdb-3track  t000_.3track.npz  t000_.atab  t000_.hhr  t000_.msa0.a3m  t000_.msa0.ss2.a3m  t000_.ss2
$ ls pdb-3track

any suggestion? run e2e version normal, thanks

**Comment by liuboeo** [comment]
I also ran into the same problem, besides, side chain information can not be got from e2e script

**Comment by ajian-he** [comment]
the side chain info is also missing in my e2e run

**Comment by Rusfell** [comment]
> runs normal, however, there is no pdb file out in the result folder
> 
> $ run_pyrosetta_ver.sh XXX.fasta .
> Running HHblits
> Running PSIPRED
> Running hhsearch
> Predicting distance and orientations
> Running parallel RosettaTR.py
> $ ls
> hhblits log parallel.fold.list pdb-3track t000_.3track.npz t000_.atab t000_.hhr t000_.msa0.a3m t000_.msa0.ss2.a3m t000_.ss2
> $ ls pdb-3track
> 
> any suggestion? run e2e version normal, thanks

I also ran into the same problem with my own protein, but ran with the example protein normal.

**Comment by aputron** [comment]
Have you ensured that PyRosetta has been installed correctly in the `folding` conda environment? 
If you view the `$WDIR/log/folding.stderr` file you'll probably see errors regarding PyRosetta. 
Follow the instructions [here](https://www.pyrosetta.org/downloads) to install and test the running of PyRosetta.

**Comment by lkj-keeke1** [comment]
> Have you ensured that PyRosetta has been installed correctly in the `folding` conda environment?
> If you view the `$WDIR/log/folding.stderr` file you'll probably see errors regarding PyRosetta.
> Follow the instructions [here](https://www.pyrosetta.org/downloads) to install and test the running of PyRosetta.

You are right, the `folding.stderr` told me there was no module called ''pyrosetta"😂. Thanks for your help, I'm installing it now and will try again to see if the `run_pyrosetta_ver.sh` works out this time.

**Comment by lj734999889** [comment]
I ran e2e version which was normal. But when I ran Pyrosetta version, it's wrong. The folding.stderr also told me there was no module called ''pyrosetta". However, I have installed Pyrosetta4 via conda in conda-folding folder. I just don't why it doesn't work well.
